# Entreprify Solutions

## 1. Revenue and Expense Analysis

In [1]:
# Creating variables for Revenue and expense details to receive user inpit
total_revenue = int(input(f"Total revenue for the year: $"))
COGS = int(input(f"Cost of goods sold: $"))
operational_expenses = int(input(f"Operational expenses: $"))

Total revenue for the year: $150000
Cost of goods sold: $90000
Operational expenses: $30000


### *Gross Profit*

In [2]:
# Calculate Gross profit
gross_profit = total_revenue - COGS

In [3]:
# Output the result
print(f"The Gross Profit for Entreprify for the recently ended fiscal year is ${gross_profit}")

The Gross Profit for Entreprify for the recently ended fiscal year is $60000


### *Net Income*

In [4]:
# Calculate Net income
net_income = gross_profit - operational_expenses

In [5]:
# Output the result
print(f"The Net Income for Entreprify for the recently ended fiscal year is ${net_income}")

The Net Income for Entreprify for the recently ended fiscal year is $30000


##  2. Profitability Metrics

### *Profit Margin (%)*

In [6]:
# Formula for calculating Profit Margin
profit_margin = (net_income / total_revenue) * 100

In [7]:
# Displaying the output
print(f"Profit Margin: {profit_margin}%") 

# Significance of metrics  
print("\nSignificance of Profit Margin:")  
print("This metric shows how much profit the company keeps per unit of revenue after covering all costs. For example, a 20% profit margin means that for every dollar of revenue earned, 20 cents remains as profit after all costs have been covered.")  

Profit Margin: 20.0%

Significance of Profit Margin:
This metric shows how much profit the company keeps per unit of revenue after covering all costs. For example, a 20% profit margin means that for every dollar of revenue earned, 20 cents remains as profit after all costs have been covered.


### *Operating Margin (%)*

In [8]:
# Formula for calculating operating Margin
operating_margin = (gross_profit / total_revenue) * 100 

In [9]:
# Displaying the output
print(f"Operating Margin: {operating_margin}%") 

# Significant 
print("\nSignificance of Operating Margin:")  
print("This reflects the percentage of revenue left after covering operating expenses, including the cost of goods sold and other operational costs. For example, a 40% operating margin means the company retains 40 cents of every dollar in revenue after covering operational expenses. It shows how efficiently the company is managing its core business activities.") 

Operating Margin: 40.0%

Significance of Operating Margin:
This reflects the percentage of revenue left after covering operating expenses, including the cost of goods sold and other operational costs. For example, a 40% operating margin means the company retains 40 cents of every dollar in revenue after covering operational expenses. It shows how efficiently the company is managing its core business activities.


## 3. Loan Repayment

In [10]:
# Creating variables to take user input on Loan details
P = int(input(f"Loan Amount = $"))
annual_interest_rate = float(input(f"Annual interest rate (in %) = "))/100
number_of_years = int(input(f"Total number of payments (in years) = "))

Loan Amount = $20000
Annual interest rate (in %) = 8
Total number of payments (in years) = 5


In [11]:
# Calculating monthly interest rate and total number of payments
r = annual_interest_rate / 12
n = number_of_years * 12

In [12]:
# Monthly repayment formula
monthly_payment = (P * r * (1 + r)**n) / ((1 + r)**n - 1)

In [13]:
# Displaying the calculated monthly repayment
print(f"The monthly repayment amount is ${monthly_payment:.2f}")

The monthly repayment amount is $405.53


## 4. Break-Even Analysis

In [14]:
# Creating variables to receive user input on the company's cost and price details
selling_price_unit = float(input(f"Selling Price per unit = $"))
variable_cost_per_unit = float(input(f"Variable Cost per unit = $"))
fixed_cost_per_month = int(input(f"Fixed Cost per month = $"))

Selling Price per unit = $50
Variable Cost per unit = $30
Fixed Cost per month = $10000


In [15]:
# Formula for calculating break-even point
break_even_point = fixed_cost_per_month/(selling_price_unit-variable_cost_per_unit)
print(f"The break-even point is {break_even_point} units")

The break-even point is 500.0 units


#### **Explanation of results  in terms of its operational goals**

The break-even point of **500 units** sets the minimum sales needed to avoid losses. Key goals include:  
- **Sales Target:** Sell at least 500 planners monthly to break even.  
- **Profit Strategy:** Exceeding 500 sales generates profit, guiding plans for profitability.  
- **Cost Management:** High break-even point may require cost reviews or pricing adjustments.  
- **Financial Health:** Selling at least 500 planners to meeting or surpass the break-even point ensures sustainability as both costs and expenses are covered to prevent loss.

##  5. Investment Growth

In [16]:
# Creating variable for investment parameters to receive user inputs
P = int(input(f"Annual investment amount = $"))
r = float(input(f"Annual return rate (decimal) = "))/100
t = int(input(f"Number of years = "))

Annual investment amount = $3000
Annual return rate (decimal) = 7
Number of years = 10


In [17]:
# Future value formula
future_value = P * ((1+r)**t-1)/r

In [18]:
# Present the value
print(f"The future value of the investment after {t} years is: ${future_value:,.2f}")

The future value of the investment after 10 years is: $41,449.34


#### **Alignment with Financial Goals:**

It indicates the total value of an investment after a specified period.   
Over 10 years, the company plans to invest \$3,000 annually, amounting to a total contribution of 30,000 dollars. With a 7\% annual return, this investment will grow to approximately **$41449.34**, showing the significant impact of compound interest. 
This aligns with the company’s financial goals by generating additional funds that can be used to support future projects, expand operations, or improve financial stability. The investment not only preserves the principal amount but also provides a return that can help achieve long-term business objectives.